docs/src/numerical_implementation/elliptic_solvers.md

```diff
@@ -213,28 +213,31 @@ solver.
 
 ### Implicit free surface pressure operator
 
-The implicit free surface solver solves for the free-surface, ``\eta``, in the vertically
-integrated continuity equation
+The implicit free surface solver solves for the free-surface, ``\eta(x, y, t)``, in the vertically
+integrated continuity equation:
 
 ```math
     \begin{equation}
     \label{eq:vertically-integrated-continuity}
-    \partial_t \eta + \partial_x (H \hat{u}) + \partial_y (H \hat{v}) = M \, ,
+    \partial_t \eta + \partial_x \left ( \int_{-H}^0 u \, \mathrm{d}z \right ) + \partial_y \left ( \int_{-H}^0 v \, \mathrm{d}z \right ) = M \, ,
     \end{equation}
 ```
 
-where ``M`` is some surface volume flux (e.g., terms such as precipitation, evaporation and runoff); 
-currently we assume that ``M=0``. To form a linear system that can be solved implicitly we recast
-the continuity equation into a discrete integral form
+where ``H(x, y)`` is the depth of the water column (to linear order with respect to the free surface elevation)
+and ``M`` is some surface volume flux (e.g., terms such as precipitation, evaporation and runoff); currently the
+model takes ``M = 0``.
+
+To form a linear system that can be solved implicitly we recast the continuity equation into a discrete
```

Comment:
This is really integrating over the volume.  Can we say this explicitly?

Reply:
Can you elaborate a bit more?

But yes, these are the sort of things we'd like to clarify in the PR!

Reply:
I added a bunch of clarifications.